### ISSUE TITLE
RUCSS and Minify CSS: The CSS variable value 0px is changed to 0 and breaks calc() function

### ISSUE THREAD

**Issue by NataliaDrause** [wp-media/wp-rocket#4557]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version - 3.10.4
- Used the search feature to ensure that the bug hasn’t been reported before - yes

**Describe the bug**
When a CSS variable declared as `0px` the `px` part is removed in Used CSS and calculation with `calc()` function returns a different result.
This is happening during minification process and likely related to the minification library. 

The same issue appears when only **Minify CSS** is enabled.

Issue with Minify CSS can be resolved by excluding the affected file, although this is not possible with Remove Unused CSS.
Safe-listing the stylesheet does not resolve the issue.

**To Reproduce**

1. Create an HTML element:
`<div class="my-style"></div>`
2. In _**Customize -> Additional CSS**_ add the following CSS code:
```
.my-style {
	width: 100%;
	height: 300px;
	background-color: red;
}
:root {
	--my-variable:0px;
}
.my-style{
	position: absolute;
	top:calc(50% - var(--my-variable));
	left:calc(50% - var(--my-variable));
}
```
3. Enable Remove Unused CSS and wait for the process to complete
4. Check the page in the incognito browser window - the position of the element will be shifted.

**Expected behavior**
The value `0px` needs to be preserved.

**Screenshots**
Screenshots in the ticket note: https://secure.helpscout.net/conversation/1712457153/311651/#thread-5029266622

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by mostafa-hisham** [comment]
```
:root {
	--my-variable:0px;
}
```
The value 0px is preserved in the new approach.

**Comment by piotrbak** [comment]
@mostafa-hisham I'm experiencing the opposite. Could you check it here?
https://rocketlabsqa.ovh/js-fetcher/

Search for: `:root{--my-variable:0`

**Comment by mostafa-hisham** [comment]
@piotrbak 
all variables in :root{} are preserved
![image](https://user-images.githubusercontent.com/29744252/148897965-389ac1bd-cf26-4b50-a387-b43e3a917fdf.png)
![root](https://user-images.githubusercontent.com/29744252/148898405-5a134d42-f9db-4fbd-bad9-d0fc6436ac2a.png)

**Comment by mostafa-hisham** [comment]
@piotrbak @engahmeds3ed 
the variable is preserved form the sass side but it getting remove from the plugin side 
i think we need to disable the minification in the plugin side we already do this in sass server

![image](https://user-images.githubusercontent.com/29744252/148901410-1a5e3a93-ed82-412c-b53f-3114e9a10a7d.png)

**Comment by piotrbak** [comment]
@mostafa-hisham it's inside the `In Progress` column, should I move it back to the `To do`?

**Comment by mostafa-hisham** [comment]
@piotrbak
@engahmeds3ed removed the minification from the plugin side
this should fix the issue
please check it

**Comment by piotrbak** [comment]
@mostafa-hisham It's good, thank you!

**Comment by piotrbak** [comment]
Fixed with 3.11
